src/api.rs

```diff
@@ -101,7 +102,7 @@ impl Default for Registration {
         Self {
             app_name: "".into(),
             instance_id: "".into(),
-            sdk_version: "unleash-client-rust-0.1.0".into(),
```

Comment:
* this version was out of sync with the cargo version 
* also in out backend we assume a different format `unleash-client-rust:version`
I decided to create a function capturing those requirements